### ISSUE TITLE
kubernetes_manifest documentation is incorrect

### ISSUE THREAD

**Issue by sansarspectrum** [hashicorp/terraform-provider-kubernetes#1409]
https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/manifest#field_manager

The example provided here, shouldn't have a `fields` bock inside field_manager.

### Terraform Version, Provider Version and Kubernetes Version

```
Terraform version: 1.0.2
Kubernetes provider version: 2.5.0
Kubernetes version: 1.19
```
### Terraform Configuration Files
```hcl
resource "kubernetes_manifest" "test" {
  provider = kubernetes-alpha

  manifest = {
    // ...
  }

  field_manager {
    fields = {
      # set the name of the field manager
      name = "myteam"

      # force field manager conflicts to be overridden
      force_conflicts = true
    }
  }
}
```

### Steps to Reproduce

1. configure the fields_manager block as specified by the above documentation
2. `terraform plan`

### Expected Behavior

The `fields_manger` changes should be applied

### Actual Behavior
Terraform errored out:
```
$ terraform plan
. . .
╷
│ Error: Unsupported argument
│
│   on main.tf line 38, in resource "kubernetes_manifest" "cluster_auth_map":
│   38:     fields = {
│
│ An argument named "fields" is not expected here.
```

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by alexsomesan** [comment]
Thanks for spotting this and pointing it out, @sansarspectrum. We'll get it fixed ASAP!

**Comment by jrhouston** [comment]
Fixed in #1410
